### PR TITLE
[bitnami/*] Adapt values.yaml of Elasticsearch, etcd and external-dns charts

### PR DESCRIPTION
**Pull request by pablogalegoc**
**Description of the change**

Add metadata to `values.yaml` in order to be used by [readme-generator](https://github.com/bitnami-labs/readme-generator-for-helm) to automatically generate the "Parameters" section of the `README.md`.

General rules followed:
- If `values.yaml` doesn't have a description for a key in a comment → Use `README.md` description
- If `values.yaml` does have a description for a key in a comment but the `README.md` description provides more info → Use `README.md` description
- Else → Leave the `values.yaml` description

**Benefits**

Less manual work and consistency between `values.yaml` and `README.md`.

**Additional information**

Commented keys are not picked by readme-generator so some had to be uncommented in order to appear in the README, however none of them affect the default installation of the chart, verified comparing the output before and after the changes of
```bash
helm install chart . -f values.yaml --dry-run --debug
```

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/elasticsearch/README.md [hunk 1]
```diff
@@ -50,407 +50,431 @@ $ helm delete --purge my-release
...
+| Name                             | Description                                 | Value                   |
+| -------------------------------- | ------------------------------------------- | ----------------------- |
+| `sysctlImage.enabled`            | Enable kernel settings modifier image       | `true`                  |
+| `sysctlImage.registry`           | Kernel settings modifier image registry     | `docker.io`             |
+| `sysctlImage.repository`         | Kernel settings modifier image repository   | `bitnami/bitnami-shell` |
+| `sysctlImage.tag`                | Kernel settings modifier image tag          | `10-debian-10-r127`     |
+| `sysctlImage.pullPolicy`         | Kernel settings modifier image pull policy  | `Always`                |
+| `sysctlImage.pullSecrets`        | Kernel settings modifier image pull secrets | `[]`                    |
+| `sysctlImage.resources.limits`   | The resources limits for the container      | `{}`                    |
+| `sysctlImage.resources.requests` | The requested resources for the container   | `{}`                    |
```
Comment (alemorcuq) [inline bitnami/elasticsearch/README.md:460]: This doesn't look like a `VolumePermissions` section

File: bitnami/elasticsearch/README.md [hunk 2]
```diff
@@ -50,407 +50,431 @@ $ helm delete --purge my-release
...
+| `volumePermissions.image.pullPolicy`   | Init container volume-permissions image pull policy                                                                                                       | `Always`                |
+| `volumePermissions.image.pullSecrets`  | Init container volume-permissions image pull secrets                                                                                                      | `[]`                    |
+| `volumePermissions.resources.limits`   | The resources limits for the container                                                                                                                    | `{}`                    |
+| `volumePermissions.resources.requests` | The requested resources for the container                                                                                                                 | `{}`                    |
+| `kibana.elasticsearch.hosts`           | Array containing hostnames for the ES instances. Used to generate the URL                                                                                 | `[]`                    |
```
Comment (alemorcuq) [inline bitnami/elasticsearch/README.md:475]: Why are these parameters in this section?

**Review by alemorcuq — CHANGES_REQUESTED** [review]

File: bitnami/elasticsearch/README.md (hunk 1 above)
Comment (pablogalegoc) [reply bitnami/elasticsearch/README.md:460]: Indeed, copy-paste error as always. Fixed in 58b852f. Thanks!

File: bitnami/elasticsearch/README.md (hunk 2 above)
Comment (pablogalegoc) [reply bitnami/elasticsearch/README.md]: Forgot to add the Kibana Parameters section, fixed in 80c7257. Thanks 🙂

**Review by alemorcuq — APPROVED** [review]

**Pull request merged**